README.md

```diff
@@ -71,18 +71,33 @@ relay.next(3);
 Console output:
 
 ```none
+observerA: 0
 observerA: 1
 observerA: 2
 observerB: 2
 observerA: 3
 observerB: 3
 ```
 
+The initial value of the `BehaviorRelay` is set in the constructor:
```

Comment:
You can remove all the additions from here down to the `const` change in `ReplayRelay`